src/Symfony/Component/Form/ClearableErrorsInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form;
+
+/**
+ * A form element whose errors can be cleared.
+ *
+ * @author Colin O'Dell <colinodell@gmail.com>
+ */
+interface ClearableErrorsInterface
+{
+    /**
+     * Removes all the errors of this form.
+     *
+     * @param bool $deep whether to remove errors from child forms as well
```

Comment:
Symfony code base uses capital style to comment params.

Reply:
Fixed, amended, and force-pushed :) Thanks for catching that!